.gitignore

```diff
@@ -4,4 +4,6 @@ interchaintest.test
 .idea
 /bin
 vendor
-examples/hyperspace/exported_state.json
\ No newline at end of file
+examples/hyperspace/exported_state.json
+
+go.work.sum
```

Comment:
This redundant work.sum is always incorrect vs the standard go.sum's. I can't see why it is needed since it always has issues, so ignoring should be a good way to fix constant merge conflicts since adding